pages/404.js

```diff
@@ -0,0 +1,56 @@
+import Link from "next/link";
+import Image from "next/image";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowLeft } from "@fortawesome/free-solid-svg-icons";
+import { firstErrorLetter } from "../styles/image/404page_4_1.png";
```

Comment:
We can't import images like this. Image is a single component. {} in import used to get one component from multiple components. 
Ex: lib/post.js have multiple exports. If we want `fetchPost` only instead of all then we can import it like `{fetchPost}`